modules/virt-installing-virtctl-client-web.adoc

```diff
@@ -0,0 +1,49 @@
+// Module included in the following assemblies:
+//
+// install/virt-installing-virtctl.adoc
+
+[id="virt-installing-virtctl-client-web_{context}"]
+= Installing the `virtctl` client from the web console
+
+You can download the `virtctl` client from the Red Hat Customer Portal, which is linked to in your {VirtProductName} web console in the *Command Line Tools* page.
+
+.Prerequisites
+
+* You must have an activated {product-title} subscription to access the download page on the Customer Portal. 
+
+.Procedure
+
+. Access the Customer Portal by clicking the image:../images/question-circle.png[title="Help"] icon, which is in the upper-right corner of the web console, and selecting *Command Line Tools*. 
+
+. Ensure you have the appropriate version for your cluster selected from the `Version:` list.
+
+. Download the *virtctl* client for your distribution. All downloads are in `tar.gz` format.
```

Comment:
I could see this being marked-up `*virtctl*` as something to click on in the GUI, though given the backtick usage that begins a few lines later as binary/command references, maybe it could also just be backticks here too.